lib/Elastica/Multi/MultiBuilder.php

```diff
@@ -38,16 +38,15 @@ private function buildResultSet(Response $childResponse, BaseSearch $search)
      */
     private function buildResultSets(Response $response, $searches)
     {
-        $resultSets = array();
-
         $data = $response->getData();
-        if (!isset($data['responses']) || !is_array($data['responses'])) {
-            return $resultSets;
+        if (empty($data['responses']) || !is_array($data['responses'])) {
```

Comment:
Not 100% I understand what you mean but best is to just push a new version of what you think should be done which makes it easier to discuss.